NavigationTools.py

```
@@ -19,6 +20,10 @@ class PointObject :
 
     def __init__(self, latitude = None, longitude = None) :
         error_log = []
+        if latitude == None :
+            error_log.append(f"Error: latitude was not given a value, please input a value for latitude")
+        if longitude == None :
+            error_log.append(f"Error: longitude was not given a value, please input a value for longitude")
         if not(isinstance(latitude, (float, int))) :
             error_log.append(f"Error: latitude is of the wrong type, expected float or int and got {type(latitude)}")
         if not(isinstance(longitude, (float, int))) :
```

If we handle these differently we should raise the error between them instead of adding all 4 error messages to the log.